MathLib/LinAlg/Lis/LisMatrix.h

```diff
@@ -45,6 +45,22 @@ struct SetMatrixSparsity<LisMatrix, SPARSITY_PATTERN>;
 class LisMatrix
 {
 public:
+    /// Matrix type
+    enum class MatrixType : int
```

Comment:
Yes, in ctor of LisMatrix as a default value for lis matrix type.
I can also imagine to have
```xml
<lis>
   <options> -i cg -tol 3</options>
   <matrix_type>CRS</matrix_type>
</lis>
```